sorting/controller.go

```diff
@@ -0,0 +1,30 @@
+package sorting
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/qor/qor/admin"
+)
+
+func updatePosition(context *admin.Context) {
+	if result, err := context.FindOne(); err == nil {
+		if position, ok := result.(positionInterface); ok {
+			if pos, err := strconv.Atoi(context.Request.URL.Query().Get("pos")); err == nil {
+				if MoveTo(context.GetDB(), position, pos) == nil {
+					context.Writer.Write([]byte("OK"))
+					return
+				}
+			}
+		}
+	}
+	context.Writer.Write([]byte("Error"))
+}
+
+func (s *Sorting) InjectQorAdmin(res *admin.Resource) {
```

Comment:
it was inject the qor admin struct into itself. but it is changed to inject a resource to itself now.

It is not really inject it into admin, because it is 'injected' into qor admin already when you use `admin.AddResource(&Product{})`